var/spack/repos/builtin/packages/ucc/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Ucc(AutotoolsPackage, CudaPackage):
+    """UCC is a collective communication operations API and library that is
+    flexible, complete, and feature-rich for current and emerging programming
+    models and runtimes."""
+
+    homepage = "https://openucx.github.io/ucc/"
+    url = "https://github.com/openucx/ucc/archive/refs/tags/v1.1.0.tar.gz"
+
+    # maintainers("github_user1", "github_user2")
```

Comment:
```suggestion
```

@zzzoom leaving the commented line currently confuses @spackbot. This is something we're working on, but for now it's better to not list a maintainer directive if there's no maintainers specified in the package.

Reply:
@alecbcs got it, done